### PR TITLE
[icon-layout] Fix broken icon layout 

### LINKED ISSUE #178: Icons layout is broken.
screenshot 2017-10-30 12 27 37

### PR DESCRIPTION
**Pull request by stacyk**
Fixes #178.

### CODE REVIEW

File: scss/patterns/_tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Tooltip Styles
+// ==============
+
+
+// Style a tooltip panel
+.herman-tooltip {
```
Comment (mirisuzanne) [inline scss/patterns/_tooltip.scss]: I don't think this is being used?

File: scss/previews/_icon.scss [hunk 2]
```diff
@@ -2,35 +2,47 @@
...
   [data-icon] {
     @include square('h1');
+    margin-bottom: size('gutter');
   }
 }
 
 [data-sassdoc='icon-info'] {
   font-size: size('small');
-  padding: size('half-shim') 0;
+  position: relative;
```
Comment (mirisuzanne) [inline scss/previews/_icon.scss]: Leftover from tooltips?

File: scss/previews/_icon.scss [hunk 3]
```diff
@@ -2,35 +2,47 @@
...
+
+.herman-file-extension {
+  font-size: size('xsmall');
+  font-weight: normal;
 }
 
 [data-sassdoc='icon-path'] {
   @include font-family('code');
+  font-size: size('xxsmall');
+  text-align: left;
```
Comment (mirisuzanne) [inline scss/previews/_icon.scss]: Not sure this is needed. Maybe another tooltip issue?

File: scss/previews/_icon.scss (hunk 2 above)
Comment (stacyk) [reply scss/previews/_icon.scss]: yes

File: scss/previews/_icon.scss (hunk 3 above)
Comment (stacyk) [reply scss/previews/_icon.scss]: yep

**Pull request merged**